src/ontology/mondo-edit.obo

```diff
@@ -74363,18 +74355,25 @@ subset: rare
 synonym: "malignant ovarian mixed epithelial neoplasm" EXACT [NCIT:C40090]
```

Comment:
@sabrinatoro [NCIT:40090](https://www.ebi.ac.uk/ols4/ontologies/ncit/classes?short_form=NCIT_C40090) no longer has any of the synonyms that are included here with NCIT:40090 as the source. Should those also be edited in this ticket? The [Physician Data Query ontology](https://bioportal.bioontology.org/ontologies/PDQ/?p=classes&conceptid=http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FPDQ%2FCDR0000039597) in BioPortal does include these synonyms and is mapped in NCIT:40090 so PDQ:CDR0000039597 could be the source.

Reply:
@twhetzel yes, let's update the synonyms since we are here. I can do that.